src/v2/modules/documents/validator.js

```diff
@@ -1,16 +1,29 @@
 'use strict';
 
-const Joi = require('@hapi/joi');
+const Joi = require('@hapi/joi')
+  .extend(require('@hapi/joi-date'));
+const DATE = Joi.date().format('YYYY-MM-DD');
 const validators = require('../../lib/validators');
 
+const documentSchema = Joi.object({
+  regime: Joi.string().required().valid('water'),
+  documentType: Joi.string().required().valid('abstraction_licence'),
+  versionNumber: Joi.number().integer().required().min(0),
+  documentRef: Joi.string().required(),
+  status: Joi.string().required().valid('current', 'draft', 'superseded'),
+  startDate: DATE.required(),
```

Comment:
I think there are existing items in `validators` for some of these - it would be good to have them all in one place, e.g:
```
{
  startDate : validators.START_DATE.
  endDate : validators.END_DATE,
  isTest : validators.TEST_FLAG
}
```
etc.

Reply:
done